server/runtime/src/main/java/org/infinispan/server/tasks/TaskParameter.java

```diff
@@ -0,0 +1,25 @@
+package org.infinispan.server.tasks;
+
+import org.infinispan.commons.marshall.ProtoStreamTypeIds;
+import org.infinispan.protostream.annotations.ProtoFactory;
+import org.infinispan.protostream.annotations.ProtoField;
+import org.infinispan.protostream.annotations.ProtoTypeId;
+
+/**
+ * @author Tristan Tarrant &lt;tristan@infinispan.org&gt;
+ * @since 11.0
+ **/
+@ProtoTypeId(ProtoStreamTypeIds.DISTRIBUTED_SERVER_TASK_PARAMETER)
+public class TaskParameter {
+   @ProtoField(number = 1)
+   String key;
+
+   @ProtoField(number = 2)
+   String value;
+
+   @ProtoFactory
+   public TaskParameter(String key, String value) {
```

Comment:
Class and constructor can be package-private as only used internally.

Reply:
Fixed